javascript/javascript2/week2/hyfBay.js

```diff
@@ -0,0 +1,60 @@
+  
+console.log('Script loaded');
+
+const products = getAvailableProducts();
+const productsUl = document.querySelector('section.products ul');
+console.log(productsUl);
+
+function renderProducts(products) {
+    products.forEach(product => {
+        const li = document.createElement('li');
+
+        let shipsToHTML = '';
```

Comment:
I am not sure what is shipsToHTML, can you please describe why we need it here?

Reply:
Hi I'm really sorry for my late answer , but I think it was one of the part that I copy from stackoverflow :(